runtime/src/main/java/jp/co/soramitsu/runtime/multiNetwork/connection/ConnectionPool.kt

```diff
@@ -29,6 +33,36 @@ class ConnectionPool @Inject constructor(
     private val connectionWatcher = MutableLiveData<Unit>()
 
     init {
+        connectionWatcher.switchMap {
+            val connListFlow = pool.map {
+                it.value.isConnecting.map { isConnecting ->
+                    it.value.chain to isConnecting
+                }
+            }
+            val connChainsListFlow = combine(connListFlow) { chains ->
+                val issues = chains.filter { (_, isConnecting) -> isConnecting }.map { (chain, _) ->
+                    NetworkIssueItemState(
+                        iconUrl = chain.icon,
+                        title = chain.name,
+                        type = when {
+                            chain.nodes.size > 1 -> NetworkIssueType.Node
+                            else -> NetworkIssueType.Network
+                        },
+                        chainId = chain.id,
+                        chainName = chain.name,
+                        assetId = chain.utilityAsset.id,
+                        priceId = chain.utilityAsset.priceId
+                    )
+                }
+                issues
+            }
+
+            connChainsListFlow.asLiveData(this)
+        }
+            .observeForever {
```

Comment:
It's not necessary to cast the connChainsListFlow to LiveData, you can use collect

```
this.launch {
                connChainsListFlow.collect {
                    networkStateMixin.updateNetworkIssues(it)
                }
            }
```